libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/internal/navigation/HttpClientTest.kt

```diff
@@ -0,0 +1,100 @@
+package com.mapbox.services.android.navigation.v5.internal.navigation
+
+import com.mapbox.navigator.HttpCode
+import okhttp3.Interceptor
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.After
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+
+class HttpClientTest {
+
+    companion object {
+        private const val TEST_URL = "https://test.url.com/"
+        private const val HEADER_USER_AGENT = "User-Agent"
+        private const val HEADER_ENCODING = "Accept-Encoding"
+        private const val USER_AGENT = "testAgent"
+        private const val GZIP = "gzip"
+        private const val SUCCESS_BODY = "Success test message"
+        private const val FAILURE_BODY = "Error test message"
+        private const val SUCCESS_CODE = 200
+        private const val FAILURE_CODE = 401
+    }
+
+    private val httpClient = HttpClient(USER_AGENT)
+    private val httpClientWithGzip = HttpClient(USER_AGENT, true)
+    private val mockServer = MockWebServer()
+
+    @Before
+    fun setUp() {
+        mockServer.start()
+        mockServer.url(TEST_URL)
+    }
+
+    @After
+    fun cleanUp() {
+        mockServer.shutdown()
+    }
+
+    @Test
+    fun `check success network response`() {
+        val mockResponse = buildResponse(SUCCESS_CODE, SUCCESS_BODY)
+        mockServer.enqueue(mockResponse)
+        val response = httpClient.executeMockRequest()
+
+        Assert.assertEquals(HttpCode.SUCCESS.name, response.code.name)
```

Comment:
```suggestion
        assertEquals(HttpCode.SUCCESS.name, response.code.name)
```

and `import org.junit.Assert.assertEquals`

Reply:
fixed